staging/src/k8s.io/api/testdata/HEAD/certificates.k8s.io.v1beta1.CertificateSigningRequest.json

```diff
@@ -42,26 +42,27 @@
   },
   "spec": {
     "request": "OA==",
+    "signerName": "19",
```

Comment:
@liggitt I don't remember how the test is written. Are you expecting to see these change?

Reply:
I _think_ these are expected as the `testdata` subdirectory is named `HEAD`, so I think it should reflect the latest version of the code?

Reply:
yeah, changes to API types are expected to change test data in HEAD, but not in round-tripping of data snapshots from past releases